doc/api/util.md

```diff
@@ -325,6 +325,8 @@ Inherit the prototype methods from one [constructor][] into another. The
 prototype of `constructor` will be set to a new object created from
 `superConstructor`.
 
+This mainly adds some input validation on top of
+`Object.setPrototypeOf(constructor.prototype, superConstructor.prototype)`.
```

Comment:
I think this is not complete. We are replacing it with 2 setPrototypeOf call, and here I see only one.

Reply:
@mcollina The second replacement is not part of `util.inherits()` – it was added on my request (https://github.com/nodejs/node/pull/24755#issuecomment-443444612), to match ES6-style inheritance behaviour